{
  "status": "INTERNAL_SERVER_ERROR",
  "message": "Partial success of the \"User update\" operation: User with username: 24051d54-9321-4dd2-a92f-6425d6c455be attributes were successfully updated in Cognito but enable status was not",
  "error_code": "CAP-022",
  "causes": [
    "Exception while updating user in AWS Cognito: Update enable status error"
  ]
}
